examples/inclined_plane.py

```diff
@@ -31,24 +31,24 @@
 # in steps of size dt, and optionally includes measurement noise
 
 def simulate(mu, length=2.0, phi=np.pi / 6.0, dt=0.005, noise_sigma=None):
-    T = torch.zeros(1)
-    velocity = torch.zeros(1)
-    displacement = torch.zeros(1)
-    acceleration = torch.tensor([little_g * np.sin(phi)]) - \
-        torch.tensor([little_g * np.cos(phi)]) * mu
+    T = torch.zeros(())
+    velocity = torch.zeros(())
+    displacement = torch.zeros(())
+    acceleration = torch.tensor(little_g * np.sin(phi)) - \
+        torch.tensor(little_g * np.cos(phi)) * mu
 
-    if acceleration.item() <= 0.0:             # the box doesn't slide if the friction is too large
-        return torch.tensor([1.0e5])  # return a very large time instead of infinity
+    if acceleration.numpy() <= 0.0:  # the box doesn't slide if the friction is too large
```

Comment:
Why is the `.numpy()` needed? PyTorch scalars already work in comparisons.
```py
>>> if torch.tensor(1.0) > 0:
>>>    print('this works')
this works
```

Reply:
fine by me but since it's already merged...